src/client/telemetry/index.ts

```diff
@@ -1439,11 +1439,6 @@ export interface IEventNamePropertyMapping {
          */
         action: 'displayed' | 'dismissed' | 'ok' | 'cancel' | 'doNotShowAgain';
     };
-    [Telemetry.KernelSpecNotFoundError]: {
```

Comment:
I need to add the action for this. Can you regenerate telemetry again?

Reply:
Oh damn, sorry

Reply:
Done